### PR TITLE
feat: badge component 

### PR DESCRIPTION
**Pull request by borkarsaish65**
image

Created badge component.

Badge component can be added to button. 

Can be positioned to top right using a class.

Can also be used with icons and avatar. 

created status badge for avatars which can be positioned depending upon the avatar size.

### CODE REVIEW

File: component/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.bg-secondary {
+    color: var(--white);
+    background-color: var(--secondary-color);
+}
+
+.bg-danger {
+    color: var(--white);
+    background-color: rgb(255, 67, 67);
```
Comment (NamrataRaikwar2002) [inline component/badge/badge.css]: You should always use variables for colours. You can check all of your files in this PR and use variables.

**Review by NamrataRaikwar2002 — APPROVED** [review]
Good work, some suggestions are given.

**Review by DevRaahul — APPROVED** [review]
Use variables for colors and rest of the code looks fine to me

**Review by Divya70 — APPROVED** [review]
good to go

File: component/badge/badge.css (hunk 1 above)
Comment (borkarsaish65) [reply component/badge/badge.css]: Yes thanks I will fix it.

**Review by Sindhu983 — COMMENTED** [review]
Looks cool.

**Pull request merged**